_posts/2019-06-06-NeuralODEs.markdown

```diff
@@ -0,0 +1,388 @@
+---
+layout: post
+title:  "Neural ODEs"
+date:   2019-06-06 10:00:00 -0400
+categories: neural-nets
+author: luca
+blurb: "Neural ODEs are neural network models which generalize standard layer to layer propagation to continuous depth models. Starting from the observation that the forward propagation in neural networks is equivalent to one step of discretation of an ODE, we can construct and efficiently train models by via ODEs. On top of providing a novel family of architectures, notably for invertible density models and continuous time series, neural ODEs also provide a memory efficiency gain in supervised learning tasks. In this curriculum, we will go through all the background topics necessary to understand these models. At the end, you should be able to implement neural ODEs and apply them to different tasks."
+feedback: true
+---
+
+This guide would not have been possible without the help and feedback from many people. 
+
+Special thanks to Prof. Joan Bruna and his class at NYU, [Mathematics of Deep Learning](https://github.com/joanbruna/MathsDL-spring19), and to Cinjon Resnick, who introduced me to DFL and helped me putting up this guide.
+
+Thank you to Avital Oliver, Matt Johnson, Dougal MacClaurin, David Duvenaud, and Ricky Chen for useful contributions to this guide.
+
+Thank you to Tinghao Li, Chandra Prakash Konkimalla, Manikanta Srikar Yellapragada, Shan-Conrad Wolf, Deshana Desai, Yi Tang, Zhonghui Hu for helping me prepare the notes.
+
+Finally, thank you to all my fellow students who attended the recitations and provided valuable feedback.
+
+<div class="deps-graph">
+<iframe class="deps" src="/assets/nodes-deps.svg" width="200"></iframe>
+<div>Concepts used in Neural ODEs. Click to navigate.</div>
+</div>
+
+# Why
+
+Neural ODEs are neural network models which generalize standard layer to layer propagation to continuous depth models. Starting from the observation that the forward propagation in neural networks is equivalent to one step of discretation of an ODE, we can construct and efficiently train models by via ODEs. On top of providing a novel family of architectures, notably for invertible density models and continuous time series, neural ODEs also provide a memory efficiency gain in supervised learning tasks.
+
+In this curriculum, we will go through all the background topics necessary to understand these models. At the end, you should be able to implement neural ODEs and apply them to different tasks.
+
+<br />
+
+# Common resources:
+
+1. (Süli & Mayers): [An Introduction to Numerical Analysis](https://www.cambridge.org/core/books/an-introduction-to-numerical-analysis/FD8BCAD7FE68002E2179DFF68B8B7237#)
```

Comment:
For every numbered line, add a period at the end of them, e.g.:
	“An Introduction to Numerical Analysis” —> “An Introduction to Numerical Analysis.”

It looks a lot better (we had this experience before).

Reply:
all of these should have been fixed